website/default_config.py

```diff
@@ -76,8 +76,10 @@
 DOCS_BASE_URL = "https://picard-docs.musicbrainz.org"
 CHANGELOG_URL = "https://raw.githubusercontent.com/musicbrainz/picard/master/NEWS.md"
 FILESERVER_URL = "https://data.musicbrainz.org/pub/musicbrainz/picard"
+HELP_WITH_TRANSLATION_URL = "https://translations.metabrainz.org/projects/picard/website/"
```

Comment:
Or https://wiki.musicbrainz.org/MusicBrainz_Picard/Internationalization ? Not sure which URL we want to prefer, but I think the Wiki pages were meant  as the first step as they give more background information.

Reply:
Done in https://github.com/metabrainz/picard-website/pull/340/commits/b5623da0b3400601fadf7622b3bd1cd6e5a0ee1c

Reply:
What I find interesting is that nowhere on the wiki page is there a link to the Weblate translation server.

Reply:
> What I find interesting is that nowhere on the wiki page is there a link to the Weblate translation server.

That's an odd oversight. I think this came from the idea that there is the main [Internationalization](https://wiki.musicbrainz.org/Internationalization#Translation) Wiki page with all the information about the translation server, and the project page just holding details for the project. This works if the starting point is the main page. But if you start on the Picard project page it is not as clear and you easily miss the link to the main page.

I edited the Wiki page and tried to make it a bit clearer: https://wiki.musicbrainz.org/index.php?title=MusicBrainz_Picard/Internationalization&diff=78122&oldid=77248 . 

Does this look better?

Reply:
Yes, it looks better.

Reply:
> Does this look better?

Much better IMHO.  Thanks.